rllib/execution/train_ops.py

```diff
@@ -65,6 +65,104 @@ def train_one_step(trainer, train_batch) -> Dict:
     return info
 
 
+@ExperimentalAPI
+def multi_gpu_train_one_step(trainer, train_batch) -> Dict:
+    config = trainer.config
+    workers = trainer.workers
+    local_worker = workers.local_worker()
+    policies = local_worker.policies_to_train
+    num_sgd_iter = config.get("sgd_num_iter", 1)
+    sgd_minibatch_size = config.get("sgd_minibatch_size", 0)
+
+    # Collect actual GPU devices to use.
+    if not config.get("num_gpus", 0):
```

Comment:
just asking, should we throw an error and tell users to fix the config instead??
at least print a big warning so this doesn't hide user errors silently?

Reply:
Yeah, I'm actually not sure myself, why we have to use `.get()` here. I just re-used the code from the existing component. ...

Reply:
I see.
actually now that I look at train_one_step() and TrainOneStep, and multi_gpu_train_one_step() and MultiGPUTrainOneStep, I feel like we should try to make TrainOneStep call into train_one_step() and MultiGPUTrainOneStep call into multi_gpu_train_one_step(). so that we don't have code duplication and can retire the class callables whenever.
what do you think? if you think it's a good idea, you can add a TODO, and whoever gets a bit time can make it happen.